include/openmc/mesh.h

```diff
@@ -454,6 +462,8 @@ class UnstructuredMesh : public Mesh {
   virtual std::string get_mesh_type() const override;
 
   // Overridden Methods
+  // TODO Position sample(uint64_t* seed) const=0;
```

Comment:
can this go or is there more work to do here?

Reply:
Good catch, will delete and check for other TODOs